src/browser-frontend/views/browser/chrome/tabbar/tab/tab-favicon.jsx

```diff
@@ -36,15 +34,31 @@ export default class TabFavicon extends PureComponent {
 
     if (this.props.loadState === DomainPageMetaModel.LOAD_STATES.INITIAL ||
         this.props.loadState === DomainPageMetaModel.LOAD_STATES.CONNECTING) {
-      contents = <SpinnerType1 />;
+      contents = (<FittedImage
+        src="var(--theme-tab-connecting)"
+        width="28px"
+        height="16px"
+        mode="contain"
+        className="spinner"
+        style={{ marginRight: '2px' }}
+      />);
     } else if (this.props.loadState === DomainPageMetaModel.LOAD_STATES.LOADING) {
-      contents = <SpinnerType2 />;
-    } else if (this.props.loadState === DomainPageMetaModel.LOAD_STATES.LOADED) {
+      contents = (<FittedImage
+        src="var(--theme-tab-loading)"
+        width="28px"
+        height="16px"
+        mode="contain"
+        className="spinner"
+        style={{ marginRight: '2px' }}
```

Comment:
Generally, we're trying to make a case of not using inline styles unless we need to compute them based on props. For everything else, let's use regular css. Use `styleName` instead of `className` to have access to the css modules defined in `tab-favicon.css`.